src/components/Dashboard/TodoList.vue

```diff
@@ -413,6 +413,7 @@
 
                   <!-- dropdown menu -->
                   <v-menu
+                    v-if="!isCancellableTodoItem(item)"
```

Comment:
I created dummy data for two todo tasks (annual report and AGM location change) with both the payment being pending.
![different types of payment pending](https://github.com/bcgov/business-filings-ui/assets/122301442/817067fe-90a8-47f8-91ab-173b5cbc1b47)

As you can see, only the AGM location will not have the cancel payment option since it does not have a salvageable draft. I also went for a general name here since we might have more of those in the future. We can just plug in the extra types in the function below (we're going to probably add it for Extensions 😁 ).

Reply:
Two thumbs up! 👍 👍